packages/vitest-pool-workers/src/worker/types-globals.d.ts

```diff
@@ -1,11 +1,15 @@
 // noinspection ES6ConvertVarToLetConst
 
-import type { WorkerGlobalState } from "vitest";
+import type { VitestUtils, WorkerGlobalState } from "vitest";
 
 declare global {
-	// https://github.com/vitest-dev/vitest/blob/v1.0.0-beta.5/packages/vitest/src/runtime/worker.ts#L49-L69
-	const __vitest_worker__: WorkerGlobalState & {
-		config?: { poolOptions?: { workers?: SerializedOptions } };
+	// https://github.com/vitest-dev/vitest/blob/1ec3a8b687c57153ed3d0d4777d7765c18f3cc82/packages/vitest/src/runtime/utils.ts#L24
+	const __vitest_worker__: WorkerGlobalState;
+	// https://github.com/vitest-dev/vitest/blob/1ec3a8b687c57153ed3d0d4777d7765c18f3cc82/packages/vitest/src/runtime/moduleRunner/moduleRunner.ts#L86
+	const __vitest_mocker__: VitestUtils & {
+		moduleRunner: {
+			import: (id: string) => Promise<Record<string, unknown>>;
+		};
```

Comment:
Are these APIs designed to be public or are we reaching into the internals here?